src/pegin/components/create/BtcInputAmount.vue

```diff
@@ -245,17 +246,22 @@ export default defineComponent({
           && amount.lte(maxValue);
     };
 
+    function getOptionsData() {
+      calculateTxFee();
+      context.emit('get-pegin-quotes');
+    }
+
     const bitcoinAmountModel = computed({
       get() {
         return bitcoinAmount.value;
       },
       set(amount: string) {
+        clearTimeout(timeOutId.value);
         bitcoinAmount.value = amount;
         const amountInSats = new SatoshiBig(amount, 'btc');
         if (isValidAmount(amountInSats)) {
           setBtcAmount(amountInSats);
-          calculateTxFee();
-          context.emit('get-pegin-quotes');
+          timeOutId.value = setTimeout(getOptionsData, 500) as unknown as number;
```

Comment:
We could try using `window.setTimeout` and `window.clearTimeout` instead of this type assertion.